package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "defaults": {
     "omniSharp": "1.34.7",
-    "razor": "1.0.0-alpha3-20190819.3"
+    "razor": "1.0.0-alpha3-3.1.0-preview3.19563.3"
```

Comment:
new versioning scheme?

Reply:
Kind of ya. We moved where we build the Razor VSCode extension pieces into AspNetCore-Tooling and as part of that the versioning changed a lot.